### PR TITLE
Update to calendar year 2023

### PR DESCRIPTION
**Pull request by hcorson-dosch-usgs**
This PR updates the site to calendar year 2023

### Changes
* Re-run pipeline for calendar year 2023
  * This require a couple of edits to fix `sf` projection mismatches
* Adjust data file setup in site, so that test and beta sites can access different files than the '-live' versions, which control what is viewed in the live site
* Clean up the map labels placement in the grid, and adjust labels slightly to look cleaner
* Set up laptop landing view for grid to give map more space (not perfect, but an improvement)

### To test
Pull changes locally and run `npm run serve`

### Screenshots
Old laptop view:
image

New laptop view:
image

### CODE REVIEW

File: src/components/GWL.vue [hunk 1]
```diff
@@ -144,15 +147,18 @@ import { isMobile } from 'mobile-device-detect';
...
+  components: {
+    GWLmap,
+    mapLabels,
+    Legend: () => import( /* webpackPreload: true */ /*webpackChunkName: "Legend"*/ "./../components/Legend"),
+    authorship: () => import( /* webpackPreload: true */ /*webpackChunkName: "section"*/ "./../components/Authorship")
+  },
+  data() {
     return {
       publicPath: process.env.BASE_URL, // this is need for the data files in the public folder, this allows the application to find the files when on different deployment roots
+      vueTier: process.env.VUE_APP_TIER, // this is used to determine the file path suffix for data files
```
Comment (cnell-usgs) [inline src/components/GWL.vue:159]: 👍🏼

**Review by cnell-usgs — APPROVED** [review]
This looks a lot better, nice to get more screen space for the map

**Comment by hcorson-dosch-usgs** [comment]
Great! I'll push the '-live' files to S3, commit the indicator files, and then merge

**Comment by hcorson-dosch-usgs** [comment]
> Great! I'll push the '-live' files to S3, commit the indicator files, and then merge

Actually, I'd like to push to beta before updating the prod site, so will merge first w/o updating the '-live' files

**Pull request merged**